### PR TITLE
Remove note about open sourcing Motoko

### PR DESCRIPTION
**Pull request by olaszakos**
Since Motoko has already been open sourced, we can remove this line.

Requested by Elizabeth Yang

### CODE REVIEW

**Review by crusso — APPROVED** [review]
Shall we instead add a link to this repo?

File: doc/modules/language-guide/pages/motoko.adoc [hunk 1]
```diff
@@ -18,7 +18,6 @@ Check back regularly to try new features and see what's changed.
 The {proglang} programming language is a new, modern, type-sound language designed for developers who want to build the next generation of apps and services to run directly on the internet.
 Motoko is specifically designed to support the unique features of the Internet Computer and to provide a familiar yet robust programming environment.
 As a new language {proglang} is constantly evolving with support for new features and improvements.
-Keep checking back for updates and for the announcement of the {proglang} becoming available as an open source project.
```
Comment (crusso) [inline doc/modules/language-guide/pages/motoko.adoc:21]:
```suggestion
The Motoko compiler is now https://github.com/dfinity/motoko[open source]  and released under the Apache 2.0 license. Contributions are welcome.

```

Comment (crusso) [reply doc/modules/language-guide/pages/motoko.adoc:21]: @olaszakos shall we add this?

Comment (olaszakos) [reply doc/modules/language-guide/pages/motoko.adoc:21]: Yeah, thank you!

**Comment by crusso** [comment]
We're having trouble merging forks because CI doesn't complete. Re-creating this PR as #2628

**Pull request closed without merging**